### ISSUE TITLE
When fragments are exploded, highlighter is not highlighting upper storeys properly anymore

### ISSUE THREAD

**Issue by maidi29** [ThatOpen/engine_components#420]
### Describe the bug 📝

All fragments that are translated by exploder are not properly selectable by highlighter anymore.

### Reproduction ▶️

https://platform.thatopen.com/app

### Steps to reproduce 🔢

1. Go to https://platform.thatopen.com/app and load ifc
2. Explode
3. Hover over the exploded elements

### System Info 💻

```shell
System:
    OS: macOS 14.5
    CPU: (11) arm64 Apple M3 Pro
    Memory: 100.80 MB / 18.00 GB
Binaries:
    Node: 20.10.0 - ~/.nvm/versions/node/v20.10.0/bin/node
    npm: 10.2.3 - ~/.nvm/versions/node/v20.10.0/bin/npm
Browsers:
    Chrome: 126.0.6478.56
```

### Used Package Manager 📦

npm

### Validations ✅

- [X] Read the [docs](https://docs.thatopen.com/intro).
- [X] Check that there isn't [already an issue](https://github.com/ThatOpen/engine_components/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a repository issue and not a framework-specific issue. For example, if it's a THREE.js related bug, it should likely be reported to [mrdoob/threejs](https://github.com/mrdoob/three.js) instead.
- [X] Check that this is a concrete bug. For Q&A join our [Community](https://people.thatopen.com/).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by HoyosJuan** [comment]
Hey @maidi29! The platform app is using an older version of the library. Does this also happen to you in the latest version? You can take the [template](https://github.com/ThatOpen/engine_templates) as a starting point.

**Comment by maidi29** [comment]
Yes, in my app I use Version 2 of engine_components and it's the same issue there.

**Comment by agviegas** [comment]
Solved from version `@thatopen/components@2.1.3` and `@thatopen/components-front@2.1.4`. I'm publishing later today. Let me know otherwise!

**Comment by maidi29** [comment]
Works perfectly now! Thanks a lot 🥇 ❤️
